chrome/content/zotero/tabs.js

```diff
@@ -244,6 +246,124 @@ var Zotero_Tabs = new function () {
 		tab.iconBackgroundImage = icon.style.backgroundImage;
 	};
 
+	this._createLabel = (title, filter) => {
```

Comment:
`createTabMenuLabel`, and this can just be local scope without `this.` — not sure why the other private methods are using `this`.